build/build_generic_package.sh

```diff
@@ -1,12 +1,17 @@
 #!/bin/bash
-## Usage: ./build_grsec_metapackage.sh
+## Usage:
+##  ./build_package.sh securedrop-grsec
```

Comment:
Note: the build process for `securedrop-grsec` was actually moved into the new `freedomofpress/grsec` git repo.